src/components/graph/menus/network-modification-node-editor.js

```diff
@@ -5,66 +5,107 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-import React, { useState } from 'react';
-import { TextField } from '@material-ui/core';
+import React, { useEffect, useRef, useState } from 'react';
 import PropTypes from 'prop-types';
-import Grid from '@material-ui/core/Grid';
 import Button from '@material-ui/core/Button';
-import { useIntl } from 'react-intl';
-import { updateTreeNode } from '../../../utils/rest-api';
-import {
-    displayErrorMessageWithSnackbar,
-    useIntlRef,
-} from '../../../utils/messages';
+import { fetchNetworkModifications } from '../../../utils/rest-api';
+import { displayErrorMessageWithSnackbar } from '../../../utils/messages';
+import { useSelector } from 'react-redux';
+import NetworkModificationDialog from '../../dialogs/network-modifications-dialog';
+import List from '@material-ui/core/List';
+import { makeStyles } from '@material-ui/core/styles';
+import AddCircleOutlineIcon from '@material-ui/icons/AddCircleOutline';
 import { useSnackbar } from 'notistack';
+import { ModificationListItem } from './modification-list-item';
+import { ListItem } from '@material-ui/core';
+import { FormattedMessage } from 'react-intl';
+import Divider from '@material-ui/core/Divider';
 
-const NetworkModificationNodeEditor = ({ selectedNode }) => {
-    const intl = useIntl();
-    const intlRef = useIntlRef();
+const useStyles = makeStyles((theme) => ({
+    list: {
+        flexGrow: '1',
```

Comment:
You can't be sure that your parent is a flex container then this css could be useless.
Maybe we should have a parent flex div in this component which makes this consistent.

Reply:
the parent still need to grow, if the parent is not flex, the added div will not grow